packages/physics-physx/src/shape/PhysXColliderShape.ts

```
@@ -40,13 +43,15 @@ export abstract class PhysXColliderShape implements IColliderShape {
   _pxShape: any;
   /** @internal */
-  _pxGeometry: any;
   /** @internal */
   _id: number;
   /** @internal */
   _rotation: Vector3 = new Vector3();
 
   protected _physXPhysics: PhysXPhysics;
+  protected _physXGeometry: PhysXGeometry;
+  protected _pxGeometry: any;
+  protected _rotation: Vector3 = new Vector3();
```

Biome (1.9.4)

[error] Duplicate class member name "_rotation"



(lint/suspicious/noDuplicateClassMembers)